src/core/resources/nfts/nfts.ts

```diff
@@ -56,14 +55,9 @@ async function nftsQueryFunction({
   if (process.env.IS_TESTING === 'true') {
     return NFTS_TEST_DATA;
   }
-  const simpleHashSupportedChains = getSimpleHashSupportedChainNames();
-  const chains = (
-    !testnetMode ? SUPPORTED_MAINNET_CHAINS : SUPPORTED_TESTNET_CHAINS
-  )
-    .map(({ name }) => name as ChainName)
-    .filter((chainName) =>
-      simpleHashSupportedChains.includes(chainName.toLowerCase()),
-    );
+  const chains = !testnetMode
+    ? (getSimpleHashSupportedChainNames() as ChainName[])
+    : (getSimpleHashSupportedTestnetChainNames() as ChainName[]);
```

Comment:
this means that we're not filtering nfts by enabled chains from settings anymore?

Reply:
oh wait, do we even filter by enabled chain?